src/daemon/properties/mod.rs

```diff
@@ -52,12 +78,16 @@ impl fmt::Display for PropertiesCommandDetails {
 #[serde(rename_all = "snake_case")]
 #[serde(tag = "type")]
 pub enum StorablePropertiesCommand {
+    Initialise,
     UpgradeTo { krill_version: KrillVersion },
 }
 
 impl fmt::Display for StorablePropertiesCommand {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
+            Self::Initialise => {
+                write!(f, "create properties")
```

Comment:
Same comment as elsewhere about whether this should say "create" or "initialise".

Reply:
fixed